github/Requester.py

```diff
@@ -188,8 +188,6 @@ def requestMultipartAndCheck(self, verb, url, parameters=None, headers=None, inp
         return self.__check(*self.requestMultipart(verb, url, parameters, headers, input))
 
     def requestBlobAndCheck(self, verb, url, parameters=None, headers=None, input=None):
```

Comment:
I am more inclined to add a keyword argument `cnx` to `requestBlobAndCheck` here